packages/captp/package.json

```diff
@@ -30,6 +30,9 @@
   },
   "scripts": {
     "build": "exit 0",
+    "clean": "tsc --build jsconfig.build.json --clean",
```

Comment:
This is fine, but it may be good to manually verify that this works. The last Endo publish didn’t clean up after itself and I’m suspicious. I did not look deeply into why. It may be a problem higher up the foodchain, like lerna not running postpack.

Reply:
when I ran `yarn clean` it didn't remove the generated files. blocker? seems out of scope to me.

Reply:
Let’s remove the ineffective script and proceed. The work around for now is an additional step in the release process that involves nuking files that are not checked in. That carries some risk that I’ll bear going forward, until we find a better way.

Reply:
I haven't removed it yet because, from what I've read, it sometimes works. It's just very cautious to never delete if some files have been touched.

Reply:
This is not a blocker.